step-templates/pingdom-create-uptime-check.json

```diff
@@ -0,0 +1,240 @@
   # Create new check\n\n    $apiParameters = @{}\n    $apiParameters.Add(\"name\", $pingdomCheckName)\n    $apiParameters.Add(\"host\", $pingdomCheckTarget)\n    $apiParameters.Add(\"type\", $pingdomCheckType)\n    If ($pingdomCheckIntervalMinutes -ne $null) {\n        $apiParameters.Add(\"resolution\", $pingdomCheckIntervalMinutes)\n    }\n    If (-Not [string]::IsNullOrEmpty($pingdomCheckContactIds)) {\n        $apiParameters.Add(\"contactids\", $pingdomCheckContactIds)\n    }\n    If ($pingdomCheckSendToEmail -ne $null) {\n        $apiParameters.Add(\"sendtoemail\", $pingdomCheckSendToEmail)\n    }\n    If ($pingdomCheckSendNotificationWhenDown -ne $null) {\n        $apiParameters.Add(\"sendnotificationwhendown\", $pingdomCheckSendNotificationWhenDown)\n    }\n    If ($pingdomCheckNotifyAgainEvery -ne $null) {\n        $apiParameters.Add(\"notifyagainevery\", $pingdomCheckNotifyAgainEvery)\n    }\n    If ($pingdomCheckNotifyWhenBackUp -ne $null) {\n        $apiParameters.Add(\"notifywhenbackup\", $pingdomCheckNotifyWhenBackUp)\n    }\n    If (-Not [string]::IsNullOrEmpty($pingdomCheckTags)) {\n        $apiParameters.Add(\"tags\", $pingdomCheckTags)\n    }\n    If (-Not [string]::IsNullOrEmpty($pingdomCheckHttpUrl)) {\n        $apiParameters.Add(\"url\", $pingdomCheckHttpUrl)\n    }\n    If ($pingdomCheckHttpEncryptionEnabled -ne $null) {\n        $apiParameters.Add(\"encryption\", $pingdomCheckHttpEncryptionEnabled)\n    }\n    If ($pingdomCheckHttpTargetPort -ne $null) {\n        $apiParameters.Add(\"port\", $pingdomCheckHttpTargetPort)\n    }\n    If (-Not [string]::IsNullOrEmpty($pingdomCheckAuth)) {\n        $apiParameters.Add(\"auth\", $pingdomCheckAuth)\n    }\n    If (-Not [string]::IsNullOrEmpty($pingdomCheckShouldContain)) {\n        $apiParameters.Add(\"shouldcontain\", $pingdomCheckShouldContain)\n    }\n    If (-Not [string]::IsNullOrEmpty($pingdomCheckShouldNotContain)) {\n        $apiParameters.Add(\"shouldnotcontain\", $pingdomCheckShouldNotContain)\n    }\n    If (-Not [string]::IsNullOrEmpty($pingdomCheckPostData)) {\n        $apiParameters.Add(\"postdata\", $pingdomCheckPostData)\n    }\n\n    If ($apiParameters.Count -gt 0) {\n        $queryString = \"\"\n        $apiParameters.Keys | ForEach-Object { \n            $queryString += ($_ + \"=\" + [Web.HttpUtility]::HtmlEncode($apiParameters.Item($_)) + \"&\")  \n        }\n        $queryString = $queryString.Substring(0, $queryString.Length - 1)\n        $url += \"?$queryString\"\n    }\n\n    Write-Host \"Creating new uptime check: $url\"\n    Try {\n        $response = Invoke-RestMethod -Uri $url -Method Post -ContentType \"application/json\" -Credential $credential -Headers $headers\n        Write-Host \"Successfully added uptime check in Pingdom: Name = \" + ($response.check.name) + \", Id = \" + ($response.check.id)\n    } Catch {\n        Write-Error \"Error occured when adding uptime check in Pingdom: \" $_\n    }\n\n    Write-Host \"End PingdomCreateUptimeCheck\"\n}",
```

Comment:
It is actually called:
`$checkExists = $response.checks | Where-Object { $_.name -eq $pingdomCheckName } | Test-Any`